### PR TITLE
fixes #584 indicating there are other requirements

### LINKED ISSUE #584: AIA results page should mention non-scalable requirements
The AIA results page only shows the Appendix C requirements in the section Requirements Specific to Impact Level: X. It could be misleading as there are no indication that there are other requirements (section 6) to comply with.

The AIA results page should include a mention about the non-scalable requirements. Such as security, consulting legal, etc.

### PR DESCRIPTION
**Pull request by MrDeshaies**
fixes #584

### CODE REVIEW

File: src/plugins/en.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "navigateSectionLabel": "Navigate to a Specific Page (Out of 13)",
     "linkProjectText": "Link to GitHub project repository",
     "linkProjectAnchor": "https://github.com/canada-ca/aia-eia-js",
-    "linkDirective": "http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592#appC",
+    "linkDirective": "http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592",
```
Comment (nschonni) [inline src/plugins/en.json:22]: Looks like the anchor link is still used on the French side

File: src/components/Obligations.vue [hunk 2]
```diff
@@ -14,11 +14,13 @@
         <h3>{{ requirement.title }}</h3>
         <p>{{ requirement.elements[score[3] - 1].text }}</p>
       </div>
-      <p class="row">
-        <a :href="$t('linkDirective', locale)" target="_blank">
+      <div class="row">
+        <h3>{{ $t("otherRequirementsTitle", locale) }}</h3>
+        <p>{{ $t("otherRequirements", locale) }}</p>
```
Comment (nschonni) [inline src/components/Obligations.vue:19]: Not sure if you just want to wrap the whole text in the link or just pull merge the 2 paragraphs together

**Review by nschonni — APPROVED** [review]
Don't feel strongly either way with the commented lines, and didn't look at the rendered results 😉

File: src/plugins/en.json (hunk 1 above)
Comment (MrDeshaies) [reply src/plugins/en.json:22]: see that's why we do code reviews :)  thanks, I removed the anchor in French now.

File: src/components/Obligations.vue (hunk 2 above)
Comment (MrDeshaies) [reply src/components/Obligations.vue:19]: Looks good like this I think.
image

**Pull request merged**